tests/group_ops.rs

```diff
@@ -165,7 +165,7 @@ fn rotate_all() -> Result<(), IronOxideErr> {
         USER_PASSWORD,
         &Default::default(),
     )?;
-    let creator_sdk = ironoxide::initialize(&device)?;
+    let creator_sdk = ironoxide::initialize(&device.clone().into())?;
```

Comment:
Why is it necessary to `clone()` here before the `.into()`, when the call below to `initialize_check_rotation()` doesn't need a clone?

Reply:
Good catch.
It was because I was calling the `.into()` twice, which needed ownership of the device, so I had to clone it. I've fixed it now so that `device` has the `.into()` called on it when it's created, so then I can pass references to `initialize()` without needing to clone.